src/test/java/org/prebid/server/bidder/loopme/LoopmeBidderTest.java

```diff
@@ -0,0 +1,188 @@
+package org.prebid.server.bidder.loopme;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.iab.openrtb.request.Banner;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.request.Native;
+import com.iab.openrtb.request.Video;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import org.junit.jupiter.api.Test;
+import org.prebid.server.VertxTest;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderCall;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.HttpResponse;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+import org.prebid.server.proto.openrtb.ext.request.loopme.ExtImpLoopme;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Function;
+
+import static java.util.Collections.singletonList;
+import static java.util.function.Function.identity;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.banner;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.video;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.xNative;
+
+public class LoopmeBidderTest extends VertxTest {
+
+    public static final String ENDPOINT_URL = "https://test.endpoint.com";
+
+    private LoopmeBidder loopmeBidder = new LoopmeBidder(ENDPOINT_URL, jacksonMapper);
+
```

Comment:
Please check example of MetaxBidderTest
We need few more test cases.
1) creationShouldFailOnInvalidEndpointUrl
2) makeHttpRequestsShouldReturnExpectedHeaders
3) makeHttpRequestsShouldUseCorrectUri
4) makeHttpRequestsShouldHaveImpIds
5) makeHttpRequestsShouldMakeOneRequestForAllImps